frontend/src/app/facilitySelect/WithFacility.tsx

```diff
@@ -1,47 +1,63 @@
-import React from "react";
-import { useDispatch, useSelector, connect } from "react-redux";
+/* eslint-disable graphql/template-strings */
```

Comment:
```suggestion
```
Why is this necessary? It doesn't look like there are any graphql template strings in this file

Reply:
I think this can be removed, it was previously added.